app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -0,0 +1,12 @@
+
+child :content_facet => :content_attributes do
+  extends 'katello/api/v2/content_facet/base'
+
+  node do |content_facet|
+    version = content_facet.content_view_version
+    {
+      :content_view_version => version.version,
+      :content_view_version_id => version.id
```

Comment:
its computed information that isn't needed when viewing lots of hosts.  It isn't even needed in the UI or cli it just seemed like useful information.